app/src/views/home/home_screen.tsx

```diff
@@ -37,7 +35,7 @@ export default function HomeScreen({
     navigateToFriends,
   } = useHomeScreenViewModel(user, firestoreCtrl, navigation);
 
-  const [showFilterOptions, setShowFilterOptions] = useState(false);
+  //const [showFilterOptions, setShowFilterOptions] = useState(false);
```

Comment:
I thought as this is an ongoing feature not to delete it but you're right if we need it for a next feature we'll just add it back as it will lack. Thanks for noticing !